src/store/singing.ts

```diff
@@ -251,6 +273,96 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
     },
   },
 
+  ADD_NOTE: {
+    mutation(state, { note }: { note: Note }) {
+      if (state.score) {
+        const notes = [...state.score.notes].concat(note).sort((a, b) => {
+          return a.position < b.position ? -1 : 1;
+        });
```

Comment:
ソートの比較関数は、同じ値のときは0を返す必要があるので、
`return a.position - b.position;`
とした方が良いかなと思います！

また、concatは非破壊なので、配列はコピーしなくても問題ないと思います！
```suggestion
        const notes = state.score.notes.concat(note).sort((a, b) => {
          return a.position - b.position;
        });
```

Reply:
@sigprogramming 
ありがとうございます！
ソート＆配列確認いたします！
(mutationでやるべきかどうかも)

Reply:
修正ありがとうございます！
mutationですが、個人的には問題ないかなと思います…！

Reply:
指摘点修正済み・いったんResolveいたします